Shadow copy 7.0 not experimental (#25561)
dotnet/AspNetCore.Docs: aspnetcore/release-notes/aspnetcore-6.0.md

### Shadow copying in IIS

> Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](host-and-deploy/iis/advanced?view=aspnetcore-7.0#shadow-copy).

Author: ```suggestion
Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true #shadow-copy).
```

> Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).

Author: ```suggestion
Shadow copy is in preview now but will be supported when ASP.NET Core 7.0 is released. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```

> Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true #shadow-copy).

Author: ```suggestion
Shadow copy is supported in ASP.NET Core 7 and later. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).
```

> Shadow copy is now in preview but will be supported when ASP.NET Core 7.0 is released. For more information, see [Shadow copy](xref:host-and-deploy/iis/advanced?view=aspnetcore-7.0&preserve-view=true#shadow-copy).

Author: @ldillonel @adityamandaleeka  I updated it to say 
Shadow copy is now in preview but will be supported when ASP.NET Core 7.0 is released.
I could that to
Shadow copy is now in preview but **is expected*** to be supported when ASP.NET Core 7.0 is released.
